librz/include/rz_bin_dwarf.h

```diff
@@ -679,8 +679,8 @@ typedef struct {
 } RzBinDwarfAttrDef;
 
 typedef struct {
-	ut64 length;
```

Comment:
Any particular reason why you moved it?

Reply:
yes, because the struct is having fields that are positioned the opposite than the string one. tbh, this can be reverted back, but i still think is more correct to have this in this way since they are used in a big union.

Reply:
No problem then.